.circleci/config.yml

```diff
@@ -9,10 +9,13 @@ defaults: &defaults
     TERRAGRUNT_VERSION: v0.21.11
     PACKER_VERSION: 1.5.1
     GOLANG_VERSION: 1.13
-    K8S_VERSION: v1.10.0  # Same as EKS
+    K8S_VERSION: v1.15.0  # Same as EKS
+    MINIKUBE_VERSION: v1.9.2
     HELM_VERSION: v3.1.1
     KUBECONFIG: /home/circleci/.kube/config
 
+setup_minikube: &setup_minikube
```

Comment:
These changes are already in master... Do you need to rebase?

Reply:
Yes